src/pages/index.page.jsx

```diff
@@ -6,56 +6,69 @@ export default function Home() {
   return (
     <Layout pageTitle="Home">
       <div className="home">
-        <div className="home-top">
-          <div className="home-top__text-column">
-            <h1>Accessibility Cheatsheets for Components</h1>
-            <p className="home-top__top-description">
-              Intro copy goes here, such as: Build accessible components more
-              accurately and in less time with these easy-to-read cheatsheets
-              based on WCAG and ARIA best practices. Check back often as
-              we&#39;re adding new components frequently!
-            </p>
-          </div>
-          <div>
-            <img
-              className="home-top__top-image"
-              src="/home-header-image.svg"
-              alt="Illustration of dog surrounded by accessible components"
-            />
+        <div className="home-wrapper">
+          <div className="home-top">
+            <div className="home-top__text-column">
+              <h1>Accessibility Cheatsheets for Components</h1>
```

Comment:
It looks like this heading uses a default font weight of 700 instead of the 400 in Figma.

Reply:
Thanks for pointing that out! Should be fixed: https://github.com/sparkbox/accessible-components/pull/58/commits/9896b147b5b0c3715d332169aec76ae2ffef6776

Reply:
> Looks like this `<h1>` needs line height added for mobile and desktop.

In Figma, it just said "Auto" for the line-height until I clicked on that, and it changed to `34px`. I didn't realize I could click on it and get a number! 🤦‍♀️

Also that's fixed here: https://github.com/sparkbox/accessible-components/pull/58/commits/a0d86f80b4b7ccd7aeac37d5ee1ca362433b0ab7